plans/feat-input-size-output-and-preview.md

```
@@ -0,0 +1,57 @@
+# 出力解像度の入力追従 & 開始前プレビュー
+
+- Issue: https://github.com/iq3-run/crop-vcam/issues/8
+- User Prompt: `docs/20260812_追加仕様.txt` 参照
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)